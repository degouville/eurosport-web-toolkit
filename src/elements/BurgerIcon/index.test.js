import React from 'react';
import { shallow } from 'enzyme';
import { BurgerIcon } from '../..';

it('<BurgerIcon /> render', () => {
  expect(shallow(<BurgerIcon onClick={() => null} />)).toMatchSnapshot();
});
