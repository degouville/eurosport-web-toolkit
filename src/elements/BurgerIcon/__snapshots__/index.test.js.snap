// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BurgerIcon /> render 1`] = `
<Styled(a)
  data-test="burger-menu"
  onClick={[Function]}
>
  <Styled(div) />
</Styled(a)>
`;
